src/view/global-search.twig

```diff
@@ -0,0 +1,21 @@
+{% set pageType = 'global-search' %}
+{% extends "base-template.twig" %}
+{% block title %}'{{ term }}' - {{ "Search results" | trans }} - {{ GlobalConfig.serviceName }}{% endblock %}
```

Comment:
According to the current [SEO spec for page titles and descriptions](https://github.com/NatLibFi/Skosmos/issues/1533#issuecomment-2063290242), the middle part "Search results" should be dropped. So the title should include just the search term and the service name.